library/src/main/java/com/github/mproberts/rxdatabinding/bindings/ViewDataBindings.java

```diff
@@ -98,6 +98,17 @@ public void accept(Boolean value) throws Exception {
         }, view, newValue);
     }
 
+    @BindingAdapter("android:onLongClick")
+    public static void bindAndroidOnLongClick(final View view, final Runnable listener) {
+        view.setOnLongClickListener(new View.OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View v) {
+                listener.run();
+                return true;
```

Comment:
do we always want to return true here? are there any scenarios that we would not want to handle the long click